src/Raven.Studio/typescript/models/resources/serverDashboard/storagePieChart.ts

```diff
@@ -129,15 +125,18 @@ class storagePieChart {
             arcs.select("path")
                 .transition()
                 .attrTween("d", arcTween)
-                .attr("class",d => this.colorClassScale(d.data.Database));
+                .attr("class", d => this.getDatabaseColorForPie(d.data.Database));
         } else {
             arcs.select("path")
                 .attr("d", d => arc(d as any))
-                .attr("class", d => this.colorClassScale(d.data.Database));
+                .attr("class", d => this.getDatabaseColorForPie(d.data.Database));
         }
-
     }
-   
+    
+    getDatabaseColorForPie(dbName: string): string {
+        return dbName === '<System>' ? "color-11" :
+               `color-${_.findIndex(this.pieData, d => d.data.Database === dbName) % 10 + 1}`;
```

Comment:
I wonder if we can't use color scale anyway, and avoid having this.pieData. IIRC

Reply:
The color scale is now returned in the latest commit